### PR TITLE
Updated chromedriver, updated testng, added new tests for Address book and Warranty

### PR DESCRIPTION
**Pull request by yulia-ivleva**

### CODE REVIEW

File: src/pages/AddressBook.java [hunk 1]
```diff
@@ -40,4 +86,101 @@ public String getDefaultAccountAddress() {
...
     }
+
+    public AddressBook clickCreateNewAddressLink() {
+        System.out.println("Click 'Create new address' link");
+        createNewAddressLink.click();
+        return this;
+    }
+
+    public AddressBook enterAddressName(String addressName) {
+        System.out.println("Enter new address name");
```
Comment (Zlaman) [inline src/pages/AddressBook.java]: Need to log addressName

File: src/pages/AddressBook.java [hunk 2]
```diff
@@ -40,4 +86,101 @@ public String getDefaultAccountAddress() {
...
+    }
+
+    public AddressBook selectTitle(UserTitle title) {
+        System.out.println("Select user title " + title);
+        titleDropdown.selectByVisibleText(title.getTitleText());
+        return this;
+    }
+
+    public AddressBook enterFirstName(String firstName) {
+        System.out.println("Enter the first name");
```
Comment (Zlaman) [inline src/pages/AddressBook.java]: Need to log method parameter

File: src/utils/ConfigurationUtil.java [hunk 3]
```diff
@@ -1,9 +1,10 @@
 package utils;
 
 public final class ConfigurationUtil {
-    private static final String BASE_URL = "";
-    private static final String USER_NAME = "";
-    private static final String USER_PASSWORD = "";
+    private static final String BASE_URL = "https://dev08-web-soak.demandware.net/on/demandware.store/Sites-soak-ie-Site/en_IE/";
+    private static final String BASE_URL_SHORT = "https://dev08-web-soak.demandware.net/s/soak-ie/";
+    private static final String USER_NAME = "y.ivleva@astoundcommerce.com";
```
Comment (Zlaman) [inline src/utils/ConfigurationUtil.java]: Do not commit this

File: src/pages/WarrantyPage.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    public WarrantyPage clickSearchButton() {
+        System.out.println("Clicking 'Search' button");
+        searchButton.click();
+        return this;
+    }
+
+    public String getWarrantyProducts() {
+        return warrantyProducts.getText();
```
Comment (Zlaman) [inline src/pages/WarrantyPage.java]: Can there be multiple products?

Comment (yulia-ivleva) [reply src/pages/WarrantyPage.java]: Yes, but the method gets the text from title of the block with products (not from a separate product)

**Pull request merged**